app/services/mapquest_service.rb

```diff
@@ -6,12 +6,6 @@ def self.get_location(location)
     # require 'pry';binding.pry
   end
 
-  #don't know if this will work with this API, but keeping for now just in case
```

Comment:
Removed this since it was causing simplecov to lower my percentage covered. I had this in initially from the tutorials but because I'm calling the url by hand in the services, I don't need this. It will most likely be useful as a refactor, but I think my brain is just too fried to try and shift it out at the moment. I also had misspelled symbolize and put it in multiple places, so maybe that was why I was initially not getting the get_url to work? Not sure, but caught that in the final assessment and searched for any repeats after that.